### ISSUE TITLE
vkCmdClearImage only checks VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT

### ISSUE THREAD

**Issue by eintw1ck** [KhronosGroup/MoltenVK#694]
Why is this only checking `VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT`, shouldn't it be checking for depth and stencil as well for`vkCmdClearDepthStencilImage()`?

https://github.com/KhronosGroup/MoltenVK/blob/65601660190d24d6613959e389bb0a64b999b4d5/MoltenVK/MoltenVK/Commands/MVKCmdTransfer.mm#L1058

Error from RenderDoc (using lastest MoltenVk from git) unable to `vkCmdClearDepthStencilImage()` a depth attachment due to the above assertion.

```
[mvk-error] VK_ERROR_FEATURE_NOT_PRESENT: vkCmdClearImage(): Format VK_FORMAT_D32_SFLOAT_S8_UINT cannot be cleared on this device.
```

Maybe related: https://github.com/KhronosGroup/MoltenVK/issues/487

**Comment by billhollings** [comment]
Fixed in PR #717. Please retest and close if issue is fixed.

**Comment by eintw1ck** [comment]
Seems to fix it for me.
